Framework/Crystal/src/PredictSatellitePeaks.cpp

```diff
@@ -258,12 +259,21 @@ void PredictSatellitePeaks::exec_peaks() {
   outPeaks->mutableRun().addProperty<std::vector<double>>("Offset1", offsets1, true);
   outPeaks->mutableRun().addProperty<std::vector<double>>("Offset2", offsets2, true);
   outPeaks->mutableRun().addProperty<std::vector<double>>("Offset3", offsets3, true);
-  std::vector<Peak> peaks = Peaks->getPeaks();
-  for (auto it = peaks.begin(); it != peaks.end(); ++it) {
-    auto peak = *it;
-    Kernel::Matrix<double> const peak_goniometer_matrix = peak.getGoniometerMatrix();
-    int run_number = peak.getRunNumber();
-    V3D hkl = peak.getHKL();
+
+  //std::vector<Peak> peaks = Peaks->getPeaks();
+  //for (auto it = peaks.begin(); it != peaks.end(); ++it) {
```

Comment:
Please bring back the range based `for` loop. It gives the compiler more opportunity to make your code faster.